ape-config.yaml

```diff
@@ -0,0 +1,54 @@
+name: nucypher-contracts
+contracts_folder: contracts
+
+plugins:
+  - name: solidity
+
+dependencies:
+  - name: openzeppelin
+    github: OpenZeppelin/openzeppelin-contracts
+    version: 4.8.1
+  - name: openzeppelin-upgradeable
+    github: OpenZeppelin/openzeppelin-contracts-upgradeable
+    version: 4.8.1
+
+solidity:
+  version: 0.8.17
+  import_remapping:
+    - "@openzeppelin/contracts=openzeppelin/v4.8.1"
+    - "@openzeppelin-upgradeable/contracts=openzeppelin-upgradeable/v4.8.1"
+
+deployments:
+  ethereum:
+    local:
+      - nu_token_supply: 1_000_000_000
+        pre_min_authorization: 40000000000000000000000
+        pre_min_operator_seconds: 86400  # one day in seconds
+        verify: False
+  #   rinkeby:
+  #     - nu_token: '0x78D591D90a4a768B9D2790deA465D472b6Fe0f18'
+  #       nu_token_supply: 1_000_000_000
+  #       t_staking: '0x18eFb520dA5D387982C860a64855C14C0AcADF3F'
+  #       work_lock: '0x0000000000000000000000000000000000000000'
+  #       staking_escrow: '0x6A6F917a3FF3d33d26BB4743140F205486cD6B4B'
+  #       pre_min_authorization: 40000000000000000000000
+  #       pre_min_operator_seconds: 86400  # one day in seconds
+  #       verify: True
+  #   goerli:
```

Comment:
These values are used in the scripts to deploy test networks. Can they be added back in?